src/schema/schema.ts

```diff
@@ -108,6 +109,7 @@ const rootFields: any = {
   gene_families: GeneFamilies,
   gene_family: GeneFamily,
   home_page: HomePage,
+  local_discovery_city: LocalDiscoveryCity,
```

Comment:
We’ve been using camelCase for new fields

Reply:
Not a big fan of this very specific name. Can this just be “city”?

Reply:
Do we also prefer camel for all filenames now? I was wondering about `city_data.json`

Reply:
Glad you brought this up — I went with this naming for a couple of reasons:

- It seemed to me that the notion of _city_ being used here was not necessarily a general one, because of our need to handle non-Artsy shows in the local discovery use case

- I anticipated we might have a need for other local discovery-specific concepts, as we flesh out stubbed shows, follows etc

But if that's all premature I'm happy to go with the simpler naming.

Reply:
Good question. I think it could, but don’t recall some sort of standardization.

Reply:
Yeah I get that the _current_ data that’s available is all related to local discovery, but it doesn’t always need to be in the future. Cities can be connected to other parts of our data in many ways.

Generally, viewing a root field as a special node in the graph is limiting, eg consider a new feature on an artwork page that lists shows in the city that the artist of the artwork hails from.

Does that make sense?

Reply:
We touched on this briefly during the data discussion we had but unless there's a good reason not to (backwards compatibility with existing consumers) I think it's safe to augment our existing models (City, Fair, Show, etc...), we're adding this data for the purpose of Local Discovery but it can and may be used for other features as well.

We should definitely double check for existing types but since `City` is new. We might as well.

Reply:
Yep, I'm good with `City`, and take the point about keeping root fields general. 

As for augmenting vs. new models — I wasn't sure what we would do in the case where we need to resolve shows e.g. to some place other than Gravity (since that data pipeline question is still open). 
Still getting the hang of this kind of orchestration, but all of the above sounds good to me for now 👍

Reply:
The `City` type can specify the `shows` field config, including resolver, so that’s where it would instantiate a list of `Show` entries based on sourced from elsewhere.